exchange/events_test.go

```diff
@@ -192,3 +195,296 @@ func Test_isEventAllowed(t *testing.T) {
 		})
 	}
 }
+
+func TestConvertToVastEvent(t *testing.T) {
```

Comment:
Can you add one more test case here with `GeneratedBidID` set?

Reply:
Done